qmri_mtsat/derivatives/qMRLab/fmap/sub-01_TB1map.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file and its NIfTI should be under `qmri_mtsat/derivatives/qMRLab/sub-01/fmap`. (`sub-01` missing.)